recipe/build.sh

```diff
@@ -3,17 +3,27 @@ set -euxo pipefail
 
 rm -rf build || true
 
+
 if [ ${cuda_compiler_version} != "None" ]; then
   CUDA_CMAKE_OPTIONS="-DCMAKE_CUDA_COMPILER=${CUDA_HOME}/bin/nvcc -DCMAKE_CUDA_HOST_COMPILER=${CXX} -DCUDA_ARCH_NAME=All "
   USE_CUDA=ON
+  # Remove -std=c++17 from CXXFLAGS for compatibility with nvcc
+  export CXXFLAGS="$(echo $CXXFLAGS | sed -e 's/ -std=[^ ]*//')"
+  export CFLAGS="$(echo $CFLAGS | sed -e 's/ -mtune=[^ ]*//')"
```

Comment:
Was the `-mtune` change intended?

Also do we want to remove `std` from `CFLAGS` if present?

Reply:
I can experiment to see what happens, @hmacdope added them but I can test that locally.